packages/xo-server-usage-report/src/index.js

```diff
@@ -766,7 +764,9 @@ class UsageReportPlugin {
       )
     }
 
+    const currDate = new Date().toISOString().slice(0, 10)
```

Comment:
Why isn't it defined in `dataBuilder`?

Reply:
Because it's used here https://github.com/vatesfr/xen-orchestra/pull/4799/files#diff-49681e8bd47e1571ce4a090facd48510R778